### DISCUSSION TITLE
Paid Priority ...

### DISCUSSION THREAD

**Discussion by thisguyshouldworkforus** [JuanBindez/pytubefix#99] [Q&A]
Am I able to pay/donate/sponsor to get a feature request prioritized? I'm willing to pay to get [Add support for Nebula.tv](https://github.com/JuanBindez/pytubefix/issues/60) prioritized and implemented sooner rather than later. Please let me know!

**Comment by JuanBindez** [comment]
Hello Alexander, I saw that you asked if it was possible to implement it a few months ago, but I haven't been able to see it through calmly yet, as soon as I can see it I'll let you know, but someone may come along sooner and be able to implement it, I ask you not to pay anything before We start testing, if someone implements it and has a sponsorship page, send this payment to the person who implements it first.
